lib/database.js

```diff
@@ -207,7 +211,12 @@ class Database {
   query(sql, values) {
     const data = values ? values.join(',') : '';
     this.console.debug(`${sql}\t[${data}]`);
-    return this.pool.query(sql, values);
+
+    return this.pool.query(sql, values).catch((error) => {
+      error.dbStack = error.stack;
+      Error.captureStackTrace(error);
+      throw error;
+    });
```

Comment:
@nechaido don't error instance in `this.pool.query(sql, values).catch((error) =>` contains stack?

Reply:
It's an internal psql driver stack.
From socket receive to parse SQL. Which is kinda meaningless.
You want `An Error was thrown as a result of query execution in YOU_FILE.js LINE NUMBER` instead of `An Error was thrown after parsing the SQL response in INTERNAL_FILE.js some random line`.